### PR TITLE
Feature/7

### LINKED ISSUE #7: Add user schema
Added a user.js file with a user schema.

**name:**
- set unique to true because I don't want duplicate usernames

**password:**
- no special changes

**favouriteSongs:**
- Changed type to array as there will be multiple songs
-  Set required to false as a person might not want to set their favourite songs upon sign up

### LINKED ISSUE #8: Configure and added userRouter and songRouter into separate files
Additions:
- userRouter.js
- songRouter.js

### PR DESCRIPTION
**Pull request by RomanHyde**
**Change log:**
- Added user.js model
- Added userRouter.js
- Added songRouter.js

**Fixed:**
- Routing issues defaulting to the song list content.

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Song = require('./song');
```
Comment (nmunro) [inline models/user.js:2]: Is this used?

**Review by nmunro — APPROVED** [review]
Looks good, just something to check, there appears to be an unused variable?

**Comment by RomanHyde** [comment]
This was a lot to commit all at once. Sorry, Neil!

File: models/user.js (hunk 1 above)
Comment (RomanHyde) [reply models/user.js:2]: No, it's actually not, I implemented it as part of trying to route the ObjectID from the songs but it doesn't seem to be functional at this time.

**Comment by nmunro** [comment]
Actually it's not that much XD

**Comment by nmunro** [comment]
Merge at your convenience

**Comment by RomanHyde** [comment]
I'm a newbie so I worry about these things haha.

**Pull request merged**